### ISSUE TITLE
Doesn't install on iojs 2.0.0

### ISSUE THREAD

**Issue by zaggino** [websockets/ws#495]
```
zaggino-mac:brackets-electron zaggino$ node --version
v2.0.0
zaggino-mac:brackets-electron zaggino$ npm install ws
|
> utf-8-validate@1.0.1 install /Users/zaggino/github/brackets-electron/node_modules/ws/node_modules/utf-8-validate
> node-gyp rebuild

  CXX(target) Release/obj.target/validation/src/validation.o
In file included from ../src/validation.cc:15:
In file included from ../node_modules/nan/nan.h:74:
In file included from ../node_modules/nan/nan_new.h:190:
../node_modules/nan/nan_implementation_12_inl.h:181:66: error: too many arguments to function call, expected at most 2, have 4
  return v8::Signature::New(v8::Isolate::GetCurrent(), receiver, argc, argv);
         ~~~~~~~~~~~~~~~~~~                                      ^~~~~~~~~~
/Users/zaggino/.node-gyp/2.0.0/deps/v8/include/v8.h:4188:3: note: 'New' declared here
  static Local<Signature> New(
  ^
1 error generated.
make: *** [Release/obj.target/validation/src/validation.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:269:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:169:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1009:12)
gyp ERR! System Darwin 14.3.0
gyp ERR! command "/usr/local/bin/iojs" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/zaggino/github/brackets-electron/node_modules/ws/node_modules/utf-8-validate
gyp ERR! node -v v2.0.0
gyp ERR! node-gyp -v v1.0.3
gyp ERR! not ok

> bufferutil@1.0.1 install /Users/zaggino/github/brackets-electron/node_modules/ws/node_modules/bufferutil
> node-gyp rebuild

  CXX(target) Release/obj.target/bufferutil/src/bufferutil.o
In file included from ../src/bufferutil.cc:16:
In file included from ../node_modules/nan/nan.h:74:
In file included from ../node_modules/nan/nan_new.h:190:
../node_modules/nan/nan_implementation_12_inl.h:181:66: error: too many arguments to function call, expected at most 2, have 4
  return v8::Signature::New(v8::Isolate::GetCurrent(), receiver, argc, argv);
         ~~~~~~~~~~~~~~~~~~                                      ^~~~~~~~~~
/Users/zaggino/.node-gyp/2.0.0/deps/v8/include/v8.h:4188:3: note: 'New' declared here
  static Local<Signature> New(
  ^
1 error generated.
make: *** [Release/obj.target/bufferutil/src/bufferutil.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:269:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:169:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1009:12)
gyp ERR! System Darwin 14.3.0
gyp ERR! command "/usr/local/bin/iojs" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/zaggino/github/brackets-electron/node_modules/ws/node_modules/bufferutil
gyp ERR! node -v v2.0.0
gyp ERR! node-gyp -v v1.0.3
gyp ERR! not ok
npm WARN optional dep failed, continuing utf-8-validate@1.0.1
npm WARN optional dep failed, continuing bufferutil@1.0.1
ws@0.7.1 node_modules/ws
├── options@0.0.6
└── ultron@1.0.1
```

**Comment by 3rd-Eden** [comment]
Can you confirm that master works as intended?

Arnout Kazemier

**Comment by zaggino** [comment]
Yes, I can install directly from master just fine.

**Comment by 3rd-Eden** [comment]
Allright, i'll push out a new release.

**Comment by kudos** [comment]
@3rd-Eden any sign of that new release?

**Comment by ChALkeR** [comment]
Any news on this?

**Comment by zaggino** [comment]
```"ws": "https://github.com/websockets/ws.git"``` works

**Comment by ChALkeR** [comment]
@zaggino Well, of course that works, but it's not very nice to have that in a project.

**Comment by trygve-lie** [comment]
A new release to nlm would be nice. Is it that the tests fail on Travis which stops a release?

**Comment by 3rd-Eden** [comment]
Yes, I just haven't found the time to look at it as i'm currently under some heavy deadlines that have priority over this.

**Comment by 3rd-Eden** [comment]
Released 0.7.2

**Comment by ChALkeR** [comment]
Thanks!
